docs/rspec.md

```diff
@@ -0,0 +1,96 @@
+When you provide a client to remote API, you would provide some means for its users to test their operations.
+
+Surely, they could use [webmock] to check the ultimate requests that are sent to the server. But doing this, they would inadvertedly specify not their own code, but your client's code too. What do they actually need is a means to check calls of your client's operations. This way they would back on correctness of your client, and take its interface as an endpoint.
+
+For this reason, we support a special RSpec matcher `perform_operation`. It checks, what operations are called via evil-client, and what options are used in there.
```

Comment:
I'm going to return to this place in a week. For now, I just copy-pasted your comment into the separate [issue](https://github.com/evilmartians/evil-client/issues/18)